docker-compose.yaml

```diff
@@ -1,31 +1,28 @@
+---
 # version is removed as per: https://github.com/docker/cli/pull/3445
```

Comment:
Could you remove this comment? I think it's obsoleted, rn.

Reply:
Fixed.